components/mdc/TextInput/TextField.svelte

```diff
@@ -26,6 +27,15 @@ onMount(() => {
 const focus = (node) => autofocus && node.focus()
 </script>
 
+<style>
+.required {
+  color: #da1414;
```

Comment:
This (#DA1414) is our foreground error color throughout the app. You can define it as a variable for reuse (if you prefer that structure).

The background error color is #FEEFEF.

Reply:
i was trying to reuse the same color that we're using the the cover-ui project. is this incorrect?

Reply:
The color is correct. Just wasn't sure if you wanted to define a literal here (in case we fiddle with colors later). I think there's also an error color built in to Material.

Reply:
yeah, i was trying to use that one but it looks like ui-components didn't have the same set of vars that are in cover-ui. i figured "get it work now and generic-ize it later"

Reply:
what we could do is something like we've done in 'components/Progress/_index.scss' ```.bar-color {
  @include linear-progress.bar-color(var(--progress-bar-color, var(--mdc-theme-primary)));
}```
and use something like "mdc-required-input" as a class then we can set it in our apps theme file to whatever we want with this as the default/backup.

Reply:
also see https://github.com/material-components/material-components-web/tree/master/packages/mdc-textfield they have support for required. "MDCTextFieldFoundation provides validity styling by using the :invalid and :required attributes provided by HTML5's form validation API. MDCTextFieldFoundation automatically appends an asterisk to the label text if the required attribute is set."

Reply:
Nice. I like how y'all are thinking. No urgency to generic-ize anything. I was on a product team a while back that didn't make any colors generic, so every time I asked for a change, they would push back and say it can't be done because of all the values that would need to be edited. I just didn't want a repeat of that. 😮‍💨😌